pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -176,7 +176,36 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 	// is safe.
 	if prev != nil {
 		// Check if the operator is put in the 'Network Migration' mode.
-		if _, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]; !ok {
+		val, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]
+
+		if ok && val != "start" && val != "prepare" {
+			log.Printf("invalid annotation value for migration: %s", val)
+		}
+
+		if !ok || val != "start" {
+			// Prepare migration, set status but not apply objects during preparation phase.
+			if val == "prepare" {
+				// Update Network.config.openshift.io.Status
+				status, err := r.ClusterNetworkStatus(context.TODO(), operConfig)
+				if err != nil {
+					log.Printf("Could not generate network status: %v", err)
```

Comment:
nit: log should be consistent with the status message we are setting?

Reply:
Actually, these lines are copied from https://github.com/openshift/cluster-network-operator/blob/eb90c4ed641f302f5c1b5d514257c7f0698de66a/pkg/controller/operconfig/operconfig_controller.go#L327

And I found the log and the status messages aren't consistent in other existing codes. I'm not sure whether it's a protocol we need to follow.